src/pages/candidate-info/candidate-info.html

```diff
@@ -1,86 +1,119 @@
 <ion-header>
-    <page-header></page-header>
+    <mes-header [title]='getTitle()'></mes-header>  
 </ion-header>
 
 <ion-content padding class="mes-main-content">
+    <div class="mes-header-xlg mes-view-candidate-header-padding-fix">
+        <ion-row>
+            <ion-col col-9 text-left>
+                {{getCandidateName()}}
+            </ion-col>
+            <ion-col col-3 text-right>
+                {{slotDetail.startTime}}
+            </ion-col>
+        </ion-row>
+    </div>
+    <ion-card class="mes-card">
+        <ion-card-header class="mes-header-lg">
+            Candidate information 
+        </ion-card-header>
+        <ion-card-content>
+            <ion-row>
+                <ion-col col-4 class="mes-bold">
+                    Test category
+                </ion-col>
+                <ion-col col-8>
+                    Category B
+                </ion-col>
+            </ion-row>
+            <div class="mes-divider"></div>
+            <ion-row>
+                <ion-col col-4>
+                    Slot
+                </ion-col>
+                <ion-col col-8>
+                    {{getSLotType()}}
+                </ion-col>
+            </ion-row>
+            <div class="mes-divider"></div>
+            <ion-row>
+                <ion-col col-4>
+                    Driving number
+                </ion-col>
+                <ion-col col-8>
+                    {{slotDetail.driverNumber}}
+                </ion-col>
+            </ion-row>
+            <div class="mes-divider"></div>
+            <ion-row>
+                <ion-col col-4>
+                    Application Ref.
+                </ion-col>
+                <ion-col col-8>
+                    {{slotDetail.appId}}
+                </ion-col>
+            </ion-row>
+            <div class="mes-divider"></div>
+            <ion-row>
+                <ion-col col-4>
+                    Candidate comments
+                </ion-col>
+                <ion-col col-8>
+                    <span>
+                        <img src="assets/imgs/warning.png" class="warning-icon" *ngIf="showSlotWarning(s)" />
```

Comment:
What would the `s` be in the `showSlotWarning` function parameter?

Reply:
Unused parameter - just realised it :) It's supposed to be the slotDetails object but the function actually class property ;) I'll remove it
Nice catch - thanks